### DISCUSSION TITLE
Helm chart? High Availability Version?

### DISCUSSION THREAD

**Discussion by smolinari** [postalserver/postal#1825] [Development discussions]
Hi,

Has anyone thought about or even accomplished making a Helm Chart for installing Postal into a Kubernetes cluster?

Scott

**Comment by willpower232** [comment]
It has been mentioned a few times but I haven't seen one yet

#1176 #1379 #1505 #1504

**Comment by smolinari** [comment]
@willpower232 - Thanks for the links. Much appreciated. 

It would be a shame not to get Postal onto something like k8s. From my understanding, Postal is one of the rare solutions that can support multitenancy at scale and not having it scalable i.e. not on something like k8s is kinda like tying Postal's hands behind its back. 

Or, is my perspective skewed incorrectly on this?

Scott

**Reply by willpower232** [reply]
Postal is quite scalable indeed however configuring all the IP addresses and DNS records would need more than a helm chart. You would mostly need to account for the IP addresses of the nodes or route the outgoing email traffic somehow to a fixed IP address that is beyond my knowledge.

**Reply by gigo1980** [reply]
Hi, is there still a plan to provide a helm chart for postal? 
Maybe also an additional terraform provider that can do the rest?

Is there interest in that, I also can invest a bit of time to work on it with others.
